homeassistant/components/todoist/const.py

```diff
@@ -1,35 +1,38 @@
 """Constants for the Todoist component."""
-CONF_EXTRA_PROJECTS = "custom_projects"
-CONF_PROJECT_DUE_DATE = "due_date_days"
-CONF_PROJECT_LABEL_WHITELIST = "labels"
-CONF_PROJECT_WHITELIST = "include_projects"
+from typing import Final
```

Comment:
Let's split out formatting and cleanup into a separate PR that we can review quickly and merge, to focus this PR.

Reply:
I pulled out what I could, some types come from the new library so will have to be introduced in this PR.

Here is the other PR to update types: https://github.com/home-assistant/core/pull/84210